postgis_eklentisi/postgis_eklentisi.md

```diff
@@ -0,0 +1,615 @@
+
+Yaşanan gelişmeler doğrultusunda mevcut ve geliştirilmekte olan veri tabanı yönetim sistemleri de yeni özellikleri sistemlerine entegre ederek sürece dahil olmuştur. Günümüzde oldukça popüler olan, hızlı ve yüksek güvenlikli bir ilişkisel veri tabanı yönetim sistemi olan PostgreSQL’in gelişim sürecine bakıldığında ortaya çıkışı 1977 yılına kadar dayanmaktadır. PostgreSQL arkasında büyük bir açık kaynak kod geliştirme topluluğu bulunan, SQL standartlarını ve ilişkisel veri tabanı modelini destekleyen, hızlı, güçlü ve yüksek güvenlikli bir veri tabanı yönetim sistemidir [3]. Ticari yazılımların hemen hemen tüm özelliklerini destekleyen ve aynı zamanda kendine has özellikler içeren bir sistemdir [2]. Mekansal veri yönetimi için oldukça güçlü bir çözüm sunan PostGIS eklentisi ile mekansal veriyi destekleyen ilk veri tabanı yönetim sistemlerinden biri olmuştur [4]. 
+
+PostGIS, binden fazla mekansal fonksiyonu bulunan en verimli açık kaynak kodlu mekansal veri yönetim aracıdır [5]. Çeşitli GIS yazılımları (QGIS, ArcGIS vb.) ile de entegre bir şekilde kullanılabilmektedir ve açık kaynak kodlu olduğu için gelişime açıktır. PostgreSQL kurulumundan sonra basit bir prosedür izlenerek sunucuya dahil edilen PostGIS istenilen veri tabanına basit bir SQL cümlesiyle eklenir. PostGIS eklentisinin eklenmesi ile PostgreSQL’in temel veri tipleri ve fonksiyonları genişletilerek mekansal veri tipleri ve mekansal fonksiyonlar kullanılabilir duruma getirilir. Vektör ve raster veri desteği bulunan PostGIS’te koordinat dönüşümü, uzunluk ve alan hesaplamaları, mekansal ilişkilerin sınanması gibi birçok işlem kolay ve hızlı bir şekilde gerçekleştirilebilir.
+
+Bu yazıda PostGIS kurulumundan başlanarak örnek bir vektör veri seti üzerinde kullanışlı PostGIS fonksiyonları aktarılarak kullanım örneklerine yer verilecektir.
+
+## 2. POSTGIS KURULUMU
+---
+
+PostGIS kurulumundan önce bilgisayarda PostgreSQL kurulu olmalıdır. Eğer kurulu değilse [bu adresten](https://www.postgresql.org/download/)  kullanılacak bilgisayarın işletim sistemine uygun sürüm indirilerek temel kurulum adımları izlenerek kurulum yapılabilir. Daha sonra lokal sunucuya erişmek için kullanılacak olan master kullanıcı parolası kurulum adımında belirlenir ve unutulmamalıdır. 
+
+PostgreSQL kurulumu tamamlandıktan sonra PostGIS eklentisini sunucuya dahil etmek için farklı bir prosedür izlenmektedir. PostgreSQL sunucusuna ek seçenek ve eklenti kurulmasına olanak tanıyan ve PostgreSQL kurulumu ile birlikte gelen Stack Builder uygulaması ile PostGIS eklentisi sunucuya kurulabilir. Aşağıdaki kurulum adımları izlenerek PostGIS eklentisi sunucuya dahil edilebilir [6].
+
+1. PostgreSQL kurulumu ile birlikte gelen Stack Builder uygulaması çalıştırılır ve yapılandırmak istenilen sunucu seçilerek "Sonraki" butonu ile bir sonraki adıma geçilir.
```

Comment:
Windows üzerinde kurulum şu şekilde yapılmaktadır diye başlanabilir.
Mac kullanıcıları için şu uygulama önerebiliriz; https://postgresapp.com/

Reply:
Belirttiğiniz gibi eklemeleri yaptım.